packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -100,7 +102,13 @@ class _FlutterPlatform extends PlatformPlugin {
     this.explicitObservatoryPort,
     this.host,
     this.previewDart2,
-  }) : assert(shellPath != null) {
+    @required this.trackWidgetCreation,
+  }) : assert(shellPath != null),
+       compiler = new ResidentCompiler(
```

Comment:
the resident compiler initialization code is unchanged except for passing in the trackWidgetCreation flag.